webapp/app.py

```diff
@@ -143,7 +181,7 @@ def get(self, nevra=None): # pylint: disable=arguments-differ
         yield self.flush()
 
 
-class UpdatesHandlerPost(BaseHandler):
+class UpdatesHandlerPost(PostHandler):
     """Handler for processing /updates POST requests."""
 
     @gen.coroutine
```

Comment:
Take a look at http://www.tornadoweb.org/en/stable/gen.html#tornado.gen.coroutine
`From the caller’s perspective, @gen.coroutine is similar to the combination of @return_future and @gen.engine.`
But this method doesn't return future, do it?

Reply:
If you are going to remove this decorator, make sure it works as expected. You can run simple performance test:
`siege -c 100 -r 10 -H "accept: application/json" 'http://127.0.0.1:8080/api/v1/updates POST < pack.json'`

where `pack.json` is a json with list of nevras